metricbeat/module/http/json/_meta/docs.asciidoc

```diff
@@ -45,6 +45,11 @@ Here the response from `date.jsontest.com` is returned in the configured `http_j
 
 It is required to set a namespace in the general module config section.
 
+[float]
+==== json.is_array
+With this configuration enabled the `json` metriset expects the JSON structure returned by the HTTP endpoint to be an array. Further,
```

Comment:
change metriset to metricset.

Sorry, I missed this type when I reviewed the original PR. :-/

Reply:
Typo fixed in debc0cc91.